[Install]: 'make start-all' Missing developer docker-compose.yml in root

### Installation Method

Local development (make start-all)

### What is the issue?

When running 'make start-all' 

`@docker compose up -d surrealdb` 

fails because no `docker-compose.yml` file exists in the project root.

The repository contains:

- `docker-compose.dev.yml`
- `docker-compose.full.yml`
- `docker-compose.single.yml`

—but not a default `docker-compose.yml`.

Either a `docker-compose.yml` should be added to the project root,
or the `Makefile` should be updated to reference the correct compose file.

The `installation.md` currently only explains how to start the full Docker setup, and does not address this to local dev.

Also missing ports field in `docker-compose.dev.yml` -> `surrealdb`, which cause server failed connection

### Logs

```shell
🚀 Starting Open Notebook (Database + API + Worker + Frontend)...
📊 Starting SurrealDB...
no configuration file provided: not found
make: *** [start-all] Error 1
```

### Docker Compose Configuration

```yaml

```

### Environment File

```shell

```

### System Information

 **Operating System:** macOS 15.6
**Docker version** 28.5.1
**Docker Compose version** v2.40.3-desktop.1
 **Architecture:** arm64 

### Additional Context

Suggested solution: 

Add to 'make start-all'

`@docker compose -f docker-compose.dev.yml up -d surrealdb`

and add missing port in `docker-compose.dev.yml`
```
    ports:
      - "8000:8000"
```

### Pre-submission Checklist

- [x] I tried the [Installation Assistant ChatGPT](https://chatgpt.com/g/g-68776e2765b48191bd1bae3f30212631-open-notebook-installation-assistant)
- [x] I read the relevant documentation ([Installation Guide](https://github.com/lfnovo/open-notebook/blob/main/docs/getting-started/installation.md) or [Ollama Guide](https://github.com/lfnovo/open-notebook/blob/main/docs/features/ollama.md))
- [x] I searched existing issues to see if this was already reported
- [x] I redacted all sensitive information (API keys, passwords, etc.)

--- a/Makefile
+++ b/Makefile
@@ -147,7 +147,7 @@ worker-restart: worker-stop
 start-all:
 	@echo "🚀 Starting Open Notebook (Database + API + Worker + Frontend)..."
 	@echo "📊 Starting SurrealDB..."
-	@docker compose up -d surrealdb
+	@docker compose -f docker-compose.dev.yml up -d surrealdb
 	@sleep 3
 	@echo "🔧 Starting API backend..."
 	@uv run run_api.py &

--- a/docker-compose.dev.yml
+++ b/docker-compose.dev.yml
@@ -5,6 +5,8 @@ services:
       - ./surreal_data:/mydata
     environment:
       - SURREAL_EXPERIMENTAL_GRAPHQL=true
+    ports:
+      - "8000:8000"
     command: start --log info --user root --pass root rocksdb:/mydata/mydatabase.db
     pull_policy: always
     user: root

